packages/core/src/nativeMain/kotlin/elide/core/annotations/Inline.kt

```diff
@@ -1,9 +1,9 @@
-package elide.annotations
+package elide.core.annotations
```

Comment:
these are moving to avoid split packages but are largely unused outside of core library code